Packs/VMware/Integrations/VMware/VMware.yml

```
@@ -0,0 +1,254 @@
+  - arguments:
+    - description: VM UUID of virtual machine to take snapshot of.
+      name: vm-uuid
+      required: true
+    - description: Snapshot name
+      name: name
+    - description: Snapshot description
+      name: description
+    - auto: PREDEFINED
+      defaultValue: 'True'
+      description: Snapshot the virtual machine's memory
+      name: memory
+      predefined:
+      - 'true'
+      - 'false'
+    - auto: PREDEFINED
+      defaultValue: 'False'
+      description: Quiesce guest file system (needs VMWare Tools installed).
+      name: quiesce
+      predefined:
+      - 'true'
+      - 'false'
+    description: Creates VM snapshot
+    name: vmware-create-snapshot
+  - arguments:
+    - description: Snapshot name to revert to
+      name: snapshot-name
+      required: true
+    - description: VM UUID of virtual machine to revert snapshot.
+      name: vm-uuid
+      required: true
+    description: Reverts VM to snapshot
+    execution: true
+    important:
+    - contextPath: VMWare.UUID
+      description: VM instance UUID
+      related: ''
+    - contextPath: VMWare.Snapshot
+      description: Name of the snapshot reverted to.
+      related: ''
+    name: vmware-revert-snapshot
+    outputs:
+    - contextPath: VMWare.UUID
+      description: VM instance UUID
+      type: String
+    - contextPath: VMWare.Snapshot
+      description: Name of the snapshot reverted to.
+      type: String
+  - arguments:
+    - description: VM UUID of virtual machine to get events of.
+      name: vm-uuid
+      required: true
+    - defaultValue: VmGuestRebootEvent,VmGuestShutdownEvent,VmPoweredOnEvent,VmPoweredOffEvent,VmSuspendedEvent
+      description: Type of events to get, given in CSV (i.e.  VmGuestRebootEvent,VmGuestShutdownEvent).
+      name: event-type
+    description: Gets events of VM
+    name: vmware-get-events
+  - arguments:
+    - description: VM UUID of virtual machine to change NIC state
+      name: vm-uuid
+      required: true
+    - auto: PREDEFINED
+      description: New state of the NIC to be changed
+      name: nic-state
+      predefined:
+      - connect
+      - disconnect
+      - delete
+      required: true
+    - description: Number of the NIC to be changed
+      name: nic-number
+      required: true
+    description: Changes the state of a VM NIC
+    important:
+    - contextPath: VMWare.UUID
+      description: VM instance UUID
+      related: ''
+    - contextPath: VMWare.NICState
+      description: VM NIC State (i.e. connected, disconnected, delete)
+      related: ''
+    name: vmware-change-nic-state
+    outputs:
+    - contextPath: VMWare.UUID
+      description: VM instance UUID
+      type: String
+    - contextPath: VMWare.NICState
+      description: VM NIC State (i.e. connected, disconnected, delete)
+      type: String
+  dockerimage: demisto/vmware:1.0-alpine
+  runonce: false
+  script: ''
+  subtype: python2
+  type: python
+fromversion: 6.0.0
```

i think the old one was from 5.0.0
```suggestion
fromversion: 5.0.0
```